### ISSUE TITLE
Build Login and Sign Up Views

### ISSUE THREAD

**Issue by ashatat** [ashatat/e-commerce#9]

**Comment by ashatat** [comment]
I am not sure about the HTML `GET /login endpoints` here,
the way I see it they are just routes in the react app?
I mean we should have one endpoint `/` that load the react app
then everything will be done inside the react router?

**Comment by FarahZaqout** [comment]
That is true. Too much non-react work got into my head. Will keep it in mind.

I think it's good to have the endpoints here for now, and then we will have an epic react-router issue that will include all of them.

What do  you think?

**Comment by ashatat** [comment]
yeah I think you're right
when we write the detailed issues it'll make more sense, I guess
